config/config-builder/src/key_manager_config.rs

```diff
@@ -73,19 +73,22 @@ mod test {
 
     #[test]
     fn verify_generation() {
+        let chain_id = ChainId::test();
         let json_rpc_endpoint = "http://127.12.12.12:7873";
         let rotation_period_secs = 100;
         let vault_host = "182.0.0.1:8080";
         let vault_token = "root_token";
 
         let mut key_manager_config = KeyManagerConfig::new();
+        key_manager_config.chain_id = chain_id;
         key_manager_config.json_rpc_endpoint = json_rpc_endpoint.into();
         key_manager_config.rotation_period_secs = Some(rotation_period_secs);
         key_manager_config.vault_host = vault_host.into();
         key_manager_config.vault_token = vault_token.into();
 
         let key_manager_config = key_manager_config.build().unwrap();
```

Comment:
the naming here is weird, maybe you should call the first one builder and this one is the actual config. I had to double check what we're doing here :P